src/main/java/net/datafaker/service/FakeValuesService.java

```diff
@@ -458,39 +465,42 @@ public String csv(String delimiter, char quote, boolean withHeader, int limit, S
     /**
      * Generates json based on input.
      */
-    public Json json(String... fieldExpressions) {
+    public String json(String... fieldExpressions) {
         if (fieldExpressions.length % 2 != 0) {
             throw new IllegalArgumentException("Total number of field names and field values should be even");
         }
-        Json.JsonBuilder jsonBuilder = new Json.JsonBuilder();
+
+        List<SimpleField<Object, ?>> fields = new ArrayList<>();
         for (int i = 0; i < fieldExpressions.length; i += 2) {
             final int index = i;
-            jsonBuilder.set(fieldExpressions[index], () -> fieldExpressions[index + 1]);
+            fields.add(field(fieldExpressions[index], () -> fieldExpressions[index + 1]));
         }
-        return jsonBuilder.build();
+        Schema<Object, ?> schema = Schema.of(fields.toArray(new SimpleField[0]));
+        return JSON_TRANSFORMER.generate(schema, 1);
     }
 
+
```

Comment:
Random blank insertion, when there’s already a blank line?

Reply:
Haha, I saw this, don't mind this comment at all, but I wouldn't stop a PR for it either!

Reply:
That’s fair, I agree. Nice to have reverted but not a blocker

Reply:
Accident, line deleted.